templates/master/01-master-kubelet/_base/units/kubelet.service.yaml

```diff
@@ -10,6 +10,7 @@ contents: |
   [Service]
   Type=notify
   ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
+  ExecStartPre=-/usr/sbin/restorecon /usr/local/bin/kubenswrapper /usr/bin/kubensenter
```

Comment:
Belated review but and I know this is just moving the original code but there is a giant difference between `/usr/local/bin` and `/usr/bin`: It's only the former that the MCO writes.

The files in `/usr/bin` are written by ostree, and are *immutable* at runtime and will always have their labels correct (barring corruption).

Whoever added the code originally here probably added the `-` to the ExecStart because they got EROFS, but we should have never needed to do that restorecon in the first place.

Reply:
We have to do the restorecon because MCD is laying down files after boot (about 5 minutes later). The overwritten files (that contain the same content) do not have the correct label written via MCD.

Reply:
I'm not questioning the restorecon for `/usr/local/bin` (though it is unfortunate).  I'm saying the `/usr/bin/` one should be unnecessary.